pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
-name = "xrpl-py"
-version = "3.0.0"
+name = "xrpl-py-release-test"
```

Comment:
The changes in this file are superficial. They are for testing purposes only. Production-releases of xrpl-py will only contain the semver `version` changes.

Reply:
@ckeshava I don't follow -- why are you changing the name at all?

Reply:
I had changed the name and version to ensure that I don't create Releases unintentionally. I've fixed this in 80d10619